doc-github-rules.md

```diff
@@ -0,0 +1,106 @@
+# GitHub basic process
+
+**Table of contents**
```

Comment:
do we have to replicate this file in each repo or can we just point to main repo https://github.com/opiproject/opi/pull/3

Reply:
Done, using common file under opi